roles/docker/tasks/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
 # This role contains tasks for configuring and starting docker service
 
+# XXX: figure a way to download a specific docker version
```

Comment:
The problem with this is that the package is what you want; without it you'll have to install the systemd units, etc by hand.